pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -369,12 +371,21 @@ private void initializeDevice() {
     deviceState.system.software = new HashMap<>();
     deviceState.system.software.put("firmware", "v1");
     deviceState.system.last_config = new Date(0);
-    devicePoints.extraField = configuration.extraField;
 
-    if (configuration.extraPoint != null && !configuration.extraPoint.isEmpty()) {
-      addPoint(makePoint(configuration.extraPoint,
+    // Pubber runtime options
+    if (!configuration.options.extraField.isEmpty()) {
+      devicePoints.extraField = configuration.options.extraField;
+    }
+
+    if (!configuration.options.extraPoint.isEmpty()) {
```

Comment:
semantically this should be != null, and then something else would throw an error saying "can't define an empty point name" (or just ignore that case because it's all for error checking anyway)